features/facebookImport/src/popUps/index.js

```diff
@@ -0,0 +1,17 @@
+import DataStructureInfoScreen from "./infoPopUps/dataStructureInfo.jsx";
+import ActivitiesInfoScreen from "./infoPopUps/activitiesInfo.jsx";
+import MessagesInfoScreen from "./infoPopUps/messagesInfo.jsx";
+import PicturesInfoScreen from "./infoPopUps/picturesInfo.jsx";
+import PostReactionInfoScreen from "./infoPopUps/postReactionInfo.jsx";
+import OnOffFacebookInfoScreen from "./infoPopUps/onOffFacebookInfo.jsx";
+import OffFacebookInfoScreen from "./infoPopUps/offFacebookInfo.jsx";
+
+export default {
+    "data-structure-info": DataStructureInfoScreen,
```

Comment:
naming nits: there are a bit too many "infos" here and there (in directory names and file names). Also, they're no longer infoScreen, but pop-ups.